### DISCUSSION TITLE
Does anyone know what font this is?

### DISCUSSION THREAD

**Discussion by ghost** [ryanoasis/nerd-fonts#1452] [Q&A] [answered]
<img width="866" alt="Screenshot 2023-12-07 at 10 34 53" src="https://github.com/ryanoasis/nerd-fonts/assets/92013160/56d975d0-a6f7-49af-899e-53b382dd55c2">

**Comment by MithicSpirit** [comment, marked as answer]
Reminds me of [Monaspace](https://monaspace.githubnext.com/) Krypton.

**Reply by ghost** [reply]
Looks very similar, but when used, it's not beautiful =)))) Can you suggest me some fonts?

**Reply by MithicSpirit** [reply]
I personally use [Iosevka](https://github.com/be5invis/Iosevka) with a [custom build plan](https://github.com/MithicSpirit/custom-aur/blob/main/ttc-iosevka-mithic/build-plans.toml). Considering that Iosevka is [highly](https://github.com/be5invis/Iosevka/blob/main/doc/custom-build.md) [customizable](https://typeof.net/Iosevka/customizer), I'd say that it's probably one of the best fonts out there if you're willing to put the work in, although I don't think it has characters available that look similar to Monaspace Krypton if you're really set on that look.

**Reply by Finii** [reply]
I used for years Inconsolata.
Then Cascadia Code came out and I thought it looked very ugly, much too heavy. Gave it a try anyhow, and use that ever since then, strangely.

I have terminals with varying fonts always, from tests of Nerd Fonts when I was too lazy to turn the font back. Until now no other stuck ;-)
